CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Changelog
+In this file, all the changes are listed by date (YYYY-MM-DD) and split by type of change. A link to the commit is provided.
+
+# 2021-11-04
+### Added
+- Initial draft for the CHANGELOG added
+
```

Comment:
👍  This is a good start, and I'm happy to fill this in later.

🔧 My gut feeling is that commit dates are too granular. What the user is really interested in is what changes a release brings. My thoughts are: we can start recording changes from 100.10 onwards, as that is when the toolkit was rebuilt from the ground up. I'd make the format something like this:

```md
# Changelog

## Next release

* Summary of v.next changes go here.
* Added a CHANGELOG.md

## 100.12

* Summary of changes go here.

## 100.11

* Summary of changes go here.

## 100.10

* Introduction of new toolkit architecture.

```